packages/@lwc/ssr-compiler/src/compile-js/generate-markup.ts

```diff
@@ -135,12 +139,22 @@ export function addGenerateMarkupExport(
 
     const classIdentifier = b.identifier(state.lwcClassName!);
     const renderCall = hasRenderMethod
-        ? b.callExpression(b.memberExpression(b.identifier('instance'), b.identifier('render')), [])
+        ? (b.callExpression(
+              b.memberExpression(b.identifier('instance'), b.identifier('render')),
+              []
+          ) as SimpleCallExpression & {
+              callee: MemberExpression & { property: Identifier & { name: 'render' } };
+          })
```

Comment:
These `&`ed types are pretty unclear. Can we hoist them out and name them please?